packages/ERTP/src/paymentLedger.js

```diff
@@ -248,7 +249,12 @@ export const makePaymentLedger = (
     });
   };
 
-  /** @type {MintPayment} */
+  /**
+   * Creates a new Payment containing newly minted amount.
+   *
+   * @param {Amount<K>} newAmount
+   * @returns {Payment}
```

Comment:
Oh... we don't get `Payment<K>` yet? nor `Purse<K>`? Hm... going there does seem a little complex, so I'm fine with that.